package.json

```diff
@@ -16,7 +16,20 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
-    "test": "wct -l chrome,firefox --npm",
+    "pretest": "npm install chromedriver --chromedriver-force-download",
```

Comment:
Is dit script nodig als chromedriver bij de dependencies zit?

Reply:
Ja, want de flag --chromedriver-force-download zorgt ervoor dat de binary effectief gedownload word.